Dart/src/com/jetbrains/lang/dart/folding/DartFoldingBuilder.java

```diff
@@ -258,4 +261,32 @@ private static void foldTypeArguments(@NotNull final List<FoldingDescriptor> des
       }
     }
   }
+
+  private static void foldMultilineStrings(@NotNull final List<FoldingDescriptor> descriptors,
+                                           @NotNull final PsiElement root) {
+    for (PsiElement dartString : PsiTreeUtil.findChildrenOfType(root, DartStringLiteralExpression.class)) {
```

Comment:
`PsiTreeUtil.findChildrenOfType` iterates the whole AST in the file. We also need full AST traversing to fold comments and type arguments. There's an optimization that collects both comments and type arguments in one shot. You can collect DartStringLiteralExpression in that loop too for better performance.